src/main/scala/com/fulcrumgenomics/umi/ConsensusCaller.scala

```diff
@@ -20,448 +20,166 @@
+  private val phredToAdjustedLogProbCorrect: Array[Double] = phredToAdjustedLogProbError.map(LogProbability.not)
 
-  /** Returns the next read name with format "<prefix>:<idx>", where "<prefix>" is either the supplied prefix or the
-    * longest common prefix of all read names, and "<idx>" is the 1-based consensus read index.  If no prefix was found,
-    * "CONSENSUS" is used.  If no records are given, the empty string is returned.
+  /**
+    * Returns the adjusted probability of error given a base quality. This is the probability that the base was
+    * sequenced incorrectly or that the template had the wrong base due to an error during sample-preparation
+    * post-labeling (phase 2 described above).
+    *
+    * @param qual the raw base quality as a Phred scaled number
+    * @return the LogProbability that the base is incorrect
     */
-  private def nextReadName(records: Seq[SAMRecord]): String = {
-    if (records.isEmpty) ""
-    else this.options.tag + ":" + records.head.getStringAttribute(this.options.tag)
-//    val curIdx = readIdx
-//    readIdx += 1
-//    val prefix = readNamePrefix.getOrElse(longestCommonPrefix(records.map(_.getReadName)).getOrElse("CONSENSUS"))
-//    s"$prefix:$curIdx"
+  def adjustedErrorProbability(qual: PhredScore): LogProbability = {
+    if (qual < PhredScore.MinValue) throw new IllegalArgumentException(s"Cannot adjust score lower than ${PhredScore.MinValue}")
+    else this.phredToAdjustedLogProbError(qual)
   }
 
-  /** Creates a `SAMRecord` for both ends of a pair.  If a consensus read is not given for one end of a pair, a dummy
-    * record is created.  At least one consensus read must be given.
+  /**
+    * Returns 1 - #adjustedErrorProbability
+    *
+    * @param qual the raw base quality as a Phred scaled number
+    * @return the LogProbability that the base is incorrect
     */
-  private def createAndEnqueueSamRecordPair(firstRecords: Seq[SAMRecord],
-                                            firstRead: Option[ConsensusRead],
-                                            secondRecords: Seq[SAMRecord],
-                                            secondRead: Option[ConsensusRead]): Unit = {
-    if (firstRead.isEmpty && secondRead.isEmpty) throw new IllegalArgumentException("Both consenus reads were empty.")
-    val readName = nextReadName(firstRecords++secondRecords)
-    // first end
-    createAndEnqueueSamRecord(
-      records  = firstRecords,
-      read     = firstRead.getOrElse(dummyConsensusRead(secondRead.get)),
-      readName = readName,
-      readType = FirstOfPair
-    )
-    // second end
-    createAndEnqueueSamRecord(
-      records  = secondRecords,
-      read     = secondRead.getOrElse(dummyConsensusRead(firstRead.get)),
-      readName = readName,
-      readType = SecondOfPair
-    )
+  def adjustedTruthProbability(qual: PhredScore): LogProbability = {
```

Comment:
It's not - but I figured I would leave it to be symmetric with `adjustedErrorProbability` in case we did come up with a case where we want to apply adjustments to reads prior to feeding them into the pileup - which the API still supports.